src/components/layout/navbar/Navbar.tsx

```
@@ -0,0 +1,79 @@
+		<nav className="h-20 flex border-b justify-between font-medium bg-white">
+			<Link href="/" className="flex pl-6 items-center">
+				<span className={cn("text-5xl font-semibold", poppins.className)}>
+					bitroad
+				</span>
+			</Link>
+			<div className="items-center gap-4 hidden lg:flex">
+				{navbarItems.map((item) => (
+					<NavbarItem
+						key={item.href}
+						href={item.href}
+						children={item.children}
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)